src/PrestaShopBundle/Controller/Admin/MultistoreController.php

```diff
@@ -93,6 +93,16 @@ public function header(bool $lockedToAllShopContext): Response
             $groupList = $this->entityManager->getRepository(ShopGroup::class)->findBy(['active' => true]);
         }
 
+        $associatedShops = $this->getContext()->employee->getAssociatedShops();
```

Comment:
Interesting. I am worried about one usecase: is it possible to configure an employee to "have access to ALL SHOPS" and if yes, does this code work with it?

I think this code works
- when an employee is configured to have access to some shops in particular (example: employee Mathieu F has access to shops 1, 4 and 5 but not 2 and 3)
- when an employee is configured to have access to some group of shops in particular (example: employee Mathieu F has access to groups 1 and 3 but not 2)

I am worried about the 3rd usecase

Reply:
I did not see any code that checks shop groups so there is no worries for groups. But it is a nice feature to have.